tests/ClientBuilderTest.php

```diff
@@ -210,6 +210,38 @@ public function optionsDataProvider()
             ['setErrorTypes', 0],
         ];
     }
+
+    public function testCompressionPluginAdded()
+    {
+        $builder = ClientBuilder::create(['enableCompression' => true, 'dsn' => 'https://completelyrandom@dsn.asdf/42']);
+        $builder->getClient();
+        $reflectionProperty = new \ReflectionProperty(ClientBuilder::class, 'httpClientPlugins');
+        $reflectionProperty->setAccessible(true);
+        $foundDecoderPlugin = false;
```

Comment:
Add a blank line above and below this line to improve readability. Also I would rename the variable to `$decoderPluginFound` because I beleive it's grammatically more correct (or not?)